package.json

```diff
@@ -13,7 +13,8 @@
   },
   "scripts": {
     "build": "tsc",
-    "build:fixtures": "npm run build && node lib/tools/update-fixtures",
+    "update-fixture-sources": "npm run build && node lib/tools/update-fixture-sources",
```

Comment:
Match existing form of scripts that do `verb:description`, something like `fetch:fixtures`/`generate:fixtures`/`build:fixtures`

Reply:
How is `update:fixture-sources` and `update:fixture-expected`? I find `build` a kind of unclear verb for this case, but if you prefer it I don't mind. To me the important distinction is sources vs expected output.

Reply:
Nm, now I have a flag to the existing update-fixtures script, so you can just do `npm run build:fixtures --skip-source-update`.